src/System.Private.CoreLib/shared/System/Diagnostics/Contracts/Contracts.cs

```diff
@@ -641,6 +638,82 @@ public static bool Exists<T>(IEnumerable<T> collection, Predicate<T> predicate)
         #endregion
 
         #endregion User Methods
+
+        #region Private Methods
+
+        /// <summary>
+        /// This method is used internally to trigger a failure indicating to the "programmer" that he is using the interface incorrectly.
+        /// It is NEVER used to indicate failure of actual contracts at runtime.
+        /// </summary>
+        static void AssertMustUseRewriter(ContractFailureKind kind, string contractKind)
```

Comment:
Nit: It would be nice to add explicit `private` to make this match coding conventions.